pkg/util/utils.go

```diff
@@ -252,3 +254,32 @@ func ParseInputTime(inputTime string) (time.Time, error) {
 	}
 	return time.Now().Add(-duration), nil
 }
+
+// GetGlobalOpts checks all global flags and generates the command string
+func GetGlobalOpts(c *cliconfig.RunlabelValues) string {
+	globalFlags := map[string]bool{
+		"cgroup-manager": true, "cni-config-dir": true, "conmon": true, "default-mounts-file": true,
```

Comment:
Separate PR, but could you move this to main_local.go
Add one line after each global flag gets added
```
	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.CGroupManager, "cgroup-manager", "", "Cgroup manager to use (cgroupfs or systemd, default systemd)")
        globalFlags["cgroup-manager"]=true
	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.CpuProfile, "cpu-profile", "", "Path for the cpu profiling results")
        globalFlags["cpu-profile"]=true
```

That way if a user adds a global flag, it is much more likely they will do the right thing.